submit/main/routes.py

```diff
@@ -137,8 +137,8 @@ def upload(fund_code, round):  # noqa: C901
 
             metadata["User"] = g.user.email
             current_app.logger.info(
-                "Upload successful for {fund} round {round}: {metadata}",
-                extra=dict(metadata=metadata, fund=fund_code, round=round),
+                "Upload successful for {fund} round {round}: {metadata_str}",
+                extra=dict(metadata_str=json.dumps(metadata), fund=fund_code, round=round),
```

Comment:
We are logging the user's email address here - as you can see we inject it into the `metadata` blob directly above.

This is what I talk about in my last comment - it's not that "dicts can't be logged and strings can be logged" - it's that referencing dict variables makes it easy for sensitive data to sneak in.

Let's remove the email address from here.

Reply:
Changed email address to user ID.